docs/manul_doc/Functions_and_Operators/Functions_and_Operator_Reference.md

```diff
@@ -0,0 +1,24 @@
+|  Name    |  Description    | 
```

Comment:
if not supported yet, comment out.

Reply:
functions in this table are all supported.